Clarify how to add issue numbers in PR template and CONTRIBUTING.md (#8833)
TheAlgorithms/Python: CONTRIBUTING.md

Your contribution will be tested by our [automated testing on GitHub Actions](https://github.com/TheAlgorithms/Python/actions) to save time and mental energy.  After you have submitted your pull request, you should see the GitHub Actions tests start to run at the bottom of your submission page.  If those tests fail, then click on the ___details___ button try to read through the GitHub Actions output to understand the failure.  If you do not understand, please leave a comment on your submission page and a community member will try to help.

Please help us keep our issue list small by adding `Fixes #{$ISSUE_NUMBER}` to the description of pull requests that resolve open issues.

> For example, if your pull request fixes issue #10, then you should add the following to its description:

Reviewer: ```suggestion
For example, if your pull request fixes issue #10, then please add the following to its description:
```

```
Fixes #10
```

> GitHub will use this tag to [auto-close the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) when the PR is merged.

Reviewer: ```suggestion
GitHub will use this tag to [auto-close the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if and when the PR is merged.
```
